crates/sui-core/src/consensus_handler.rs

```diff
@@ -170,35 +170,13 @@ impl<C> ConsensusHandler<C> {
     }
 
     /// Updates the execution indexes based on the provided input.
-    fn update_index_and_hash(&mut self, index: ExecutionIndices, v: &[u8]) {
-        update_index_and_hash(&mut self.last_consensus_stats, index, v)
-    }
-}
+    fn update_index(&mut self, index: ExecutionIndices) {
```

Comment:
this is trivial 2 lines now (1 real line), do you think it adds to readability to have this as a separate function anymore or can it be removed and inlined in the two places it's called?

Reply:
Good point.